pkg/detectors/mongodb/mongodb.go

```diff
@@ -61,9 +68,9 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 			if timeout == 0 {
 				timeout = defaultTimeout
 			}
-			err := verifyUri(resMatch, timeout)
-			s1.Verified = err == nil
-			if !isErrDeterminate(err) {
+			isVerified, verificationErr := verifyUri(ctx, resMatch, timeout)
+			s1.Verified = isVerified
+			if !isErrDeterminate(verificationErr) {
 				s1.SetVerificationError(err, resMatch)
```

Comment:
nit: I _think_ this should be `verificationErr`.

Reply:
It absolutely should be.